addons/account/wizard/account_automatic_entry_wizard.py

```diff
@@ -183,6 +192,14 @@ def _get_move_dict_vals_change_account(self):
             source_accounts = self.move_line_ids.mapped('account_id')
             counterpart_label = len(source_accounts) == 1 and _("Transfer from %s", source_accounts.display_name) or _("Transfer counterpart")
 
+            # We divide the amount for each account by the total balance to reflect the lines counter-parted
+            analytic_distribution = {
+                account_id: 100 * distribution_amount / counterpart_vals['balance']
+                if not counterpart_currency.is_zero(counterpart_vals['balance'])
+                else 100
```

Comment:
the indentation is weird here
```suggestion
                account_id: (
                    100 * distribution_amount / counterpart_vals['balance']
                    if not counterpart_currency.is_zero(counterpart_vals['balance'])
                    else 100
                )
```
Btw, a negative sentence is more difficult to read so:
```suggestion
                account_id: (
                    100
                    if counterpart_currency.is_zero(counterpart_vals['balance'])
                    else 100 * distribution_amount / counterpart_vals['balance']
                )
```